features/fixtures/macos/Podfile

```diff
@@ -19,5 +19,11 @@ post_install do |installer|
         end
       end
     end
+
+    if target.name == "Bugsnag" || target.name == "BugsnagNetworkRequestPlugin"
+      target.build_configurations.each do |config|
+        config.build_settings['ONLY_ACTIVE_ARCH'] = 'NO'
+      end
+    end
```

Comment:
This is not necessary - passing `ONLY_ACTIVE_ARCH=NO` to `xcodebuild` (in `features/scripts/export_mac_app.sh`) does the same thing (confirmed locally) and also doesn't slow things down when building & testing directly from Xcode.

```suggestion
```